apps/gp2-server/src/data-providers/output.data-provider.ts

```diff
@@ -433,6 +433,8 @@ export const parseContentfulGraphQLOutput = (
       .map(parseTag) ?? [];
   return {
     id: data.sys.id,
+    systemPublishedVersion: data.sys.publishedVersion ?? undefined,
```

Comment:
oh I thought we would just name the variable `systemPublishedVersion` not assign it to contentful publishedVersion 😛 

I wonder if by version they meant the like `versionsCollection.total` + 1. For example, if `versionsCollection` is empty it means it is the first version

Reply:
nah, they didn't ask for this field at all nor are they aware of our internal versioning mechanisms (i think). They asked for the publish date to be able to poll and I suggested we return our system publish version for them to track if they have the most recent resource version just in case

Reply:
ah ok! nice!